UA_LOG_SESSION_INTERNAL Causing gcc error with Debug Level and some strict flags

## Description
With UA_LOGLEVEL=100, compiling the generated single-file to an obect file with gcc (tested with gcc 12 and gcc 10) causes an error about a `%s` null value.

## Background Information / Reproduction Steps
Building the `open62541.c` with log enabled and this gcc flags, gcc returns a `format-overflow` error in `UA_LOG_SESSION_INTERNAL` macro.

```bash
 /tmp/open62541/build $ gcc -Wformat-overflow -Werror  -fdiagnostics-color=always -O3 -c -o /tmp/open62541.o open62541.c
open62541.c: In function ‘Service_AddReferences’:
open62541.c:2235:24: error: ‘%.*s’ directive argument is null [-Werror=format-overflow=]
 2235 |                        "TCP %lu\t| SecureChannel %" PRIu32 "\t| Session \"%.*s\"\t| " MSG "%.0s", \
      |                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
open62541.c:917:28: note: in definition of macro ‘UA_MACRO_EXPAND’
  917 | #define UA_MACRO_EXPAND(x) x
      |                            ^
open62541.c:2242:21: note: in expansion of macro ‘UA_LOG_SESSION_INTERNAL’
 2242 |     UA_MACRO_EXPAND(UA_LOG_SESSION_INTERNAL(LOGGER, DEBUG, SESSION, __VA_ARGS__, ""))
      |                     ^~~~~~~~~~~~~~~~~~~~~~~
open62541.c:45336:5: note: in expansion of macro ‘UA_LOG_DEBUG_SESSION’
45336 |     UA_LOG_DEBUG_SESSION(&server->config.logger, session,
      |     ^~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
```
Used CMake options:
```bash
cmake -DUA_ENABLE_AMALGAMATION=1 -DUA_LOGLEVEL=100 ..
```

## Checklist

Please provide the following information:

 - [X] open62541 Version (release number or git tag): v1.3.3
 - [ ] Other OPC UA SDKs used (client or server): 
 - [X] Operating system: Arch Linux
 - [ ] Logs (with `UA_LOGLEVEL` set as low as necessary) attached
 - [ ] Wireshark network dump attached
 - [ ] Self-contained code example attached
 - [ ] Critical issue

## Proposed Fix

```patch
diff --git a/src/server/ua_session.h b/src/server/ua_session.h
index e4e4c411..3f87f320 100644
--- a/src/server/ua_session.h
+++ b/src/server/ua_session.h
@@ -134,7 +134,7 @@ UA_Session_dequeuePublishReq(UA_Session *session);
     if(UA_LOGLEVEL <= UA_LOGLEVEL_##LEVEL) {                           \
         int nameLen = (SESSION) ? (int)(SESSION)->sessionName.length : 0; \
         const char *nameStr = (SESSION) ?                               \
-            (const char*)(SESSION)->sessionName.data : NULL;            \
+            (const char*)(SESSION)->sessionName.data : "";            \
         unsigned long sockId = ((SESSION) && (SESSION)->header.channel) ? \
             (unsigned long)(SESSION)->header.channel->connectionId : 0; \
         UA_UInt32 chanId = ((SESSION) && (SESSION)->header.channel) ?   \
```

The above patch "fixes" the compiler errors and should cause no harm. 
Does look good to you or there are any possible side effects for this?

--- a/src/server/ua_session.h
+++ b/src/server/ua_session.h
@@ -134,7 +134,7 @@ UA_Session_dequeuePublishReq(UA_Session *session);
     if(UA_LOGLEVEL <= UA_LOGLEVEL_##LEVEL) {                           \
         int nameLen = (SESSION) ? (int)(SESSION)->sessionName.length : 0; \
         const char *nameStr = (SESSION) ?                               \
-            (const char*)(SESSION)->sessionName.data : NULL;            \
+            (const char*)(SESSION)->sessionName.data : "";              \
         unsigned long sockId = ((SESSION) && (SESSION)->header.channel) ? \
             (unsigned long)(SESSION)->header.channel->connectionId : 0; \
         UA_UInt32 chanId = ((SESSION) && (SESSION)->header.channel) ?   \

